src/viur/core/bones/select.py

```diff
@@ -20,17 +20,23 @@
 """
 
 SelectBoneMultiple = list[SelectBoneValue]
-""" SelectBoneMultiple is a list of SelectBoneValue elements."""
+"""Type alias for values of a multiple SelectBone."""
+
+
+def translation_key_prefix_skeleton_bonename(bones_instance: BaseBone) -> str:
+    """Generate a translation key prefix based on the skeleton name"""
+    return f'skeleton.{bones_instance._owner.__name__.lower().removesuffix("skel")}.{bones_instance._name}.'
+
+
+def translation_key_prefix_bonename(bones_instance: BaseBone) -> str:
+    """Generate a translation key prefix based on the skeleton and bone name"""
+    return f'skeleton.{bones_instance._owner.__name__.lower().removesuffix("skel")}.{bones_instance._name}.'
```

Comment:
Why isn't this defined in `BaseBone`?

Reply:
Because only the `SelectBone` has predefined `values`